lib/common/tests/nvpair/pcmk__xe_set_bool_attr_test.c

```diff
@@ -21,7 +21,7 @@ set_attr(void **state)
     pcmk__xe_set_bool_attr(node, "a", true);
     pcmk__xe_set_bool_attr(node, "b", false);
 
-    assert_string_equal(crm_element_value(node, "a"), XML_BOOLEAN_TRUE);
+    assert_string_equal(crm_element_value(node, "a"), PCMK_VALUE_TRUE);
```

Comment:
These tests already used the constants. Is that still okay since we're testing at a slightly higher level (we're effectively testing the value of `PCMK_VALUE_TRUE` elsewhere), or switch to literals?

Reply:
This is fine too, again because our code set the value to be compared against (technically the other tests are our code themselves, but they're simulating an external caller)